vmsgen.py

```diff
@@ -1041,6 +1119,8 @@ def get_input_params():
     GENERATE_UNIQUE_OP_IDS = args.unique_operation_ids
     global TAG_SEPARATOR
     TAG_SEPARATOR = args.tag_separator
+    global REMOVE_QUERY_PARAMS_FROM_PATH
```

Comment:
I think this feature flag is unnecessary.  by default we can remove query params from path where applicable.